### PR TITLE
github action as CI

### PR DESCRIPTION
**Pull request by matbesancon**

### CODE REVIEW

**Comment by matbesancon** [comment]
The readme was updated to remove outdated elements (pointing to MPB-related projects), add the GH action badge

**Comment by matbesancon** [comment]
The other changes to the README are just breaking lines when too long, splitting on ends of sentences only

File: README.md [hunk 1]
```diff
@@ -30,15 +33,21 @@ There are several convenient ways to model MICPs in Julia and access Pavito:
...
+JuMP and Convex.jl are algebraic modeling interfaces, while MathOptInterface is a lower-level interface for providing input in raw callback or matrix form.
+Convex.jl is perhaps the most user-friendly way to provide input in conic form, since it transparently handles conversion of algebraic expressions.
+JuMP supports general nonlinear smooth functions, e.g. by using `@NLconstraint`. JuMP also supports conic modeling, but requires cones to be explicitly specified, e.g. by using `norm(x) <= t` for second-order cone constraints.
```
Comment (odow) [inline README.md:38]:
```suggestion
JuMP supports general nonlinear smooth functions, e.g. by using `@NLconstraint`. JuMP also supports conic modeling, but requires cones to be explicitly specified, e.g. by using `@constraint(model, [t; x] in SecondOrderCone())` for second-order cone constraints.
```

File: README.md [hunk 2]
```diff
@@ -11,8 +12,10 @@ For algorithms that use a conic solver instead of an NLP solver, use [Pajarito](
 ## Installation
 
 Pavito can be installed through the Julia package manager:
+
 ```
-julia> Pkg.add("Pavito")
+julia> ]
+pkg> Pkg.add("Pavito")
```
Comment (blegat) [inline README.md:18]: What's the point of entering th `Pkg` environment if you don't write `add Pavito` ?

File: README.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
-[![Build Status](https://travis-ci.com/jump-dev/Pavito.jl.svg?branch=master)](https://travis-ci.com/jump-dev/Pavito.jl) [![codecov](https://codecov.io/gh/jump-dev/Pavito.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/jump-dev/Pavito.jl)
+[![Build Status](https://github.com/jump-dev/DiffOpt.jl/workflows/CI/badge.svg)](https://github.com/jump-dev/DiffOpt.jl/actions)
```
Comment (blegat) [inline README.md:1]: DiffOpt should be Pavito

File: README.md (hunk 2 above)
Comment (matbesancon) [reply README.md:18]: correct yes, fixed in last commit

File: README.md (hunk 3 above)
Comment (matbesancon) [reply README.md:1]: fixed in last commit

**Review by blegat — APPROVED** [review]

**Pull request merged**